tests/providers/dbt/cloud/hooks/test_dbt_cloud.py

```diff
@@ -119,16 +122,37 @@ def setup_class(self):
             password=TOKEN,
         )
 
+        # Connection with `schema` parameter set
+        schema_conn = Connection(
+            conn_id=SINGLE_TENANT_CONN,
+            conn_type=DbtCloudHook.conn_type,
+            login=DEFAULT_ACCOUNT_ID,
+            password=TOKEN,
+            schema=SINGLE_TENANT_SCHEMA,
+        )
+
         db.merge_conn(account_id_conn)
         db.merge_conn(no_account_id_conn)
+        db.merge_conn(schema_conn)
 
-    def test_init_hook(self):
-        hook = DbtCloudHook()
-        assert hook.dbt_cloud_conn_id == "dbt_cloud_default"
-        assert hook.base_url == BASE_URL
+    @pytest.mark.parametrize(
+        argnames="conn_id",
+        argvalues=[ACCOUNT_ID_CONN, SINGLE_TENANT_CONN],
+        ids=["multi-tenant", "single-tenant"],
+    )
+    def test_init_hook(self, conn_id):
+        hook = DbtCloudHook(conn_id)
         assert hook.auth_type == TokenAuth
         assert hook.method == "POST"
 
+        if conn_id == ACCOUNT_ID_CONN:
+            assert hook.dbt_cloud_conn_id == ACCOUNT_ID_CONN
+            assert hook.base_url == BASE_URL
+
+        if conn_id == SINGLE_TENANT_CONN:
+            assert hook.dbt_cloud_conn_id == SINGLE_TENANT_CONN
+            assert hook.base_url == SINGLE_TENANT_URL
```

Comment:
```suggestion
    @pytest.mark.parametrize(
        argnames="conn_id, url",
        argvalues=[(ACCOUNT_ID_CONN, BASE_URL), (SINGLE_TENANT_CONN, SINGLE_TENANT_URL)],
        ids=["multi-tenant", "single-tenant"],
    )
    def test_init_hook(self, conn_id, url):
        hook = DbtCloudHook(conn_id)
        assert hook.auth_type == TokenAuth
        assert hook.method == "POST"
        assert hook.dbt_cloud_conn_id == conn_id
        assert hook.base_url == url
```
Nice! Just a suggestion to simply it a little bit.

Reply:
Much better, thanks!